src/languages/en.js

```diff
@@ -825,6 +825,9 @@ export default {
         privateDescription: 'Only people invited to this room are able to find it',
         createRoom: 'Create Room',
         roomAlreadyExists: 'A room with this name already exists',
+        pleaseEnterRoomName: 'Please enter a room name',
+        pleaseChooseWorkspace: 'Please choose a workspace',
+        pleaseChooseVisibility: 'Please choose a visibility',
```

Comment:
This can be removed. It is not possible to have no visibility set.

Reply:
Okay.